src/main/java/org/concordion/api/extension/ConcordionExtender.java

```diff
@@ -100,6 +100,14 @@
      */
     ConcordionExtender withExampleListener(ExampleListener listener);
 
+    /**
+     * Adds a filter which Concordion can apply to each specification example to decide if it should be skipped.
+     *
+     * @param filter the filter
+     * @return this
+     */
+    ConcordionExtender withExampleFilter(ExampleFilter filter);
```

Comment:
This is a new method in one of our API interfaces - so will need a version bump for Concordion to let users of this API adjust their code.